specs-go/config.go

```diff
@@ -40,13 +40,12 @@ type Process struct {
 	// Cwd is the current working directory for the process and must be
 	// relative to the container's root.
 	Cwd string `json:"cwd"`
-	// Capabilities are Linux capabilities that are kept for the container.
+	// Capabilities are Linux capabilities that are kept for the container. (this field is platform dependent)
 	Capabilities []string `json:"capabilities,omitempty" platform:"linux"`
```

Comment:
Doesn't the `platform:"linux"` tag already cover the “this field is platform dependent” idea?